astroid/brain/brain_builtin_inference.py

```diff
@@ -570,12 +570,16 @@ def infer_property(node, context=None):
     prop_func = objects.Property(
         function=inferred,
         name=inferred.name,
-        doc=getattr(inferred, "doc", None),
+        doc=getattr(inferred, "doc_node.value", None),
         lineno=node.lineno,
         parent=node,
         col_offset=node.col_offset,
     )
-    prop_func.postinit(body=[], args=inferred.args)
+    prop_func.postinit(
+        body=[],
+        args=inferred.args,
+        doc_node=getattr(inferred, "doc_node", None),
```

Comment:
Isn't it set line 573 in the constructor of Property ?

Reply:
That's the `doc` for `Property.doc`, this is for `doc_node`. I don't know why exactly but not using `getattr` crashes here, so we need to use `getattr`.